app/scripts/hglib.js

```diff
@@ -91,14 +112,19 @@ const launch = (element, config, options) => {
  *
  * @return  {Object}  Newly created HiGlass component.
  */
-export const viewer = (element, viewConfig, options) => {
+export const viewer = (element, viewConfig, options,
+  refAsPromise = false) => {
```

Comment:
```suggestion
  asPromise = false) => {
```

As a user, I have no idea what `ref` refers to. `asPromise` is shorter and to the point in my mind :)

Reply:
Yup, you're right. Changed.